Source/Core/Core/HW/EXI/EXI_DeviceMemoryCard.h

```diff
@@ -34,6 +36,9 @@ class CEXIMemoryCard : public IEXIDevice
   static void Init();
   static void Shutdown();
 
+  static std::pair<std::string /* path */, bool /* migrate */>
+  GetGCIFolderPath(int card_index, bool allow_movie_folder);
```

Comment:
What do you think of making `allow_movie_folder` an enum class for readability?

I feel like `GetGCIFolderPath(0, AllowMovieFolder::Yes)` would be a lot clearer than `GetGCIFolderPath(0, true)`.

Also, regarding the structured binding `auto` grievance, have you considered returning a proper struct? Then you could avoid auto and still make the variable const.

Reply:
It is now an enum class. I placed it outside of `CEXIMemoryCard` (but inside of `ExpansionInterface`) so that the namespace qualifiers in Movie.cpp wouldn't get too wordy.

I can't think of a good name for such a struct.

Reply:
Hmm yeah, it's hard to find a good name... `GCIFolderPath` would be the obvious candidate but it feels wrong with the `migrate` bool being in there :/ Well, I guess structured bindings are good enough.